src/shared/components/linkedIconCard/linkedIconCard.css

```diff
@@ -0,0 +1,21 @@
+.linkedIconCard {
```

Comment:
https://1drv.ms/v/s!Agaa-HaalzX5zAd4HL6BAbtvMkY4

There were very interesting width breakpoints that I think can be solved by hard-setting a font-size (maybe give it one media query downsize). I believe the FontAwesome icons are intrinsically responsive, and when you mix it with our REM stylesheet system, you get the combobulation shown. Maybe giving it padding instead of margin could help? Styles man... Good luck.